src/slidepane/SlidePane.ts

```diff
@@ -11,16 +11,18 @@ import uuid from '@dojo/core/uuid';
  * Enum for left / right alignment
  */
 export const enum Align {
-	left,
-	right
+	bottom = 'bottom',
```

Comment:
I was hoping to get clever and use string enum values as indexes when accessing CSS classes, but it didn't work out as nicely as I'd hoped.

Reply:
You would need it to be [something like this](https://www.typescriptlang.org/play/#src=const%20enum%20Align%20%7B%0A%20%20bottom%20%3D%20'bottom'%2C%0A%20%20top%20%3D%20'top'%2C%0A%20%20left%20%3D%20'left'%2C%0A%20%20right%20%3D%20'right'%0A%7D%0A%0Adeclare%20const%20css%3A%20%7B%20%5BK%20in%20Align%5D%3A%20string%3B%20%7D%3B%0A%0Alet%20direction%20%3D%20Align.bottom%3B%0A%0Aconst%20classes%20%3D%20%5B%20css%5Bdirection%5D%20%5D%3B%0A) I suspect:

```ts
const enum Align {
  bottom = 'bottom',
  top = 'top',
  left = 'left',
  right = 'right'
}

declare const css: { [K in Align]: string; };

let direction = Align.bottom;

const classes = [ css[direction] ];
```